actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -242,8 +242,9 @@ def expires_in(seconds, options = {})
       response.date = Time.now unless response.date?
     end
 
-    # Sets an HTTP 1.1 Cache-Control header of <tt>no-cache</tt> so no caching should
-    # occur by the browser or intermediate caches (like caching proxy servers).
+    # Sets an HTTP 1.1 Cache-Control header of <tt>no-cache</tt>. This means the
+    # asset will always be marked as stale, so clients must always revalidate.
```

Comment:
Can you drop the initial 'always'.
.. the asset with be marked as stale, so clients ...